client/web/compose/src/views/Admin/Pages/Edit.vue

```diff
@@ -986,6 +1000,7 @@ export default {
       const { namespaceID } = this.namespace
       return this.findLayoutsByPageID({ namespaceID, pageID: this.pageID, force: true }).then(layouts => {
         this.layouts = layouts
+        this.initialLayoutsState = JSON.parse(JSON.stringify(layouts))
```

Comment:
To keep it consistent, clone should be used/implemented for the layout.
Could you see if you can do it maybe?